sorc/build_gdas.sh

```diff
@@ -1,29 +1,29 @@
 #! /usr/bin/env bash
 set -eux
 
-script_dir=$(dirname "${BASH_SOURCE[0]}")
-cd "${script_dir}" || exit 1
-
-source gfs_utils.fd/ush/detect_machine.sh
-source gfs_utils.fd/ush/module-setup.sh
-
-# detect_machine now includes the compiler on some machines
-#   but the GDAS build script does not want it
-MACHINE_ID=$(echo "${MACHINE_ID}" | cut -d "." -f 1)
-
-case "${MACHINE_ID}" in
-    hera|orion) build_jobs=10 ;;
-    *)          build_jobs=4 ;;
-esac
-
-# Check final exec folder exists
-if [[ ! -d "../exec" ]]; then
-  mkdir ../exec
-fi
-
-cd gdas.cd
-
-WORKFLOW_BUILD="ON" BUILD_JOBS="${build_jobs}" ./build.sh -t "${MACHINE_ID}"
-
-exit $?
-
+OPTIND=1
+while getopts ":dov" option; do
```

Comment:
`-o` is being processed here but there is no case statement for it.

Also, since there are no options with arguments, you don't need the `:` case (though it doesn't hurt to leave it).

Reply:
I copied this from another script.
When it comes time to make them all consistent; such as provide more options for debug, clean, etc. it may be worthwhile to do it across the few build scripts.

Reply:
Agreed. I'm probably the one who screwed it up in whatever you copied from.